schnorr_fun/tests/frost_prop.rs

```diff
@@ -54,13 +56,18 @@ proptest! {
         let sid = b"frost-prop-test".as_slice();
         let message = Message::plain("test", b"test");
 
-        let nonces: Vec<_> = signer_indexes.iter().map(|i|
-            proto.gen_nonce(
-                &secret_shares[*i],
+        let mut nonce_rngs: Vec<ChaCha20Rng> = secret_shares.iter().map(|secret_share| {
```

Comment:
`RngAlgorithm::ChaCha` is not `Seedable` so using `ChaCha20Rng`... Doesn't seem ideal

Reply:
`ChaCha20Rng` is the right choice.